tests/php-compatibility.patch

```diff
@@ -28,7 +28,14 @@ index d1fdd9f..eab4288 100644
  tests/input/new_with_parentheses.php                  18      0
  tests/input/not_spacing.php                           8       0
  tests/input/null_coalesce_operator.php                3       0
-@@ -33,15 +35,15 @@ tests/input/superfluous-naming.php                    11      0
+@@ -27,21 +29,22 @@ tests/input/optimized-functions.php                   1       0
+ tests/input/return_type_on_closures.php               21      0
+ tests/input/return_type_on_methods.php                17      0
+ tests/input/semicolon_spacing.php                     3       0
++tests/input/single-line-array-spacing.php             3       0
+ tests/input/spread-operator.php                       6       0
+ tests/input/static-closures.php                       1       0
+ tests/input/superfluous-naming.php                    11      0
```

Comment:
what's happening in the diff here? Why so many additions?

Reply:
Ah, same stuff as in the PR before, nvm

Reply:
It's a .patch file, so that's what confuses github (and us). It's just a way patch files work.